docs/source/release_notes/release-notes.rst

```diff
@@ -50,8 +50,9 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |fixed|
+         - Remove a memory leak due to usages of Runtime#addShutdownHook to cleanup resources.
```

Comment:
Probably obvious, but may be worth saying this only affects users if they use >1 instance of AtlasDB per JVM.

Reply:
Done.